source/simulator/helper_functions.cc

```diff
@@ -1005,7 +1005,7 @@ namespace aspect
 
             //Find the local max and local min
             double min_advection_local =  std::numeric_limits<double>::max();
-            double max_advection_local = -std::numeric_limits<double>::max();
+            double max_advection_local =  std::numeric_limits<double>::min();
```

Comment:
Please revert this change. `-std::numeric_limits<double>::max()` is the correct starting point for `max_advection_local`, because we do not require the temperature or composition to be positive and so you want to start with the equivalent of minus infinity here. Numeric_limits::min() is the smallest positive number that is representable by a double so it would be a good starting point if you knew the value would be larger than (but not equal to) 0.0.